pkg/preference/preference.go

```diff
@@ -334,6 +347,16 @@ func (c *PreferenceInfo) SetConfiguration(parameter string, value string) error
 			}
 			c.OdoSettings.Timeout = &typedval
 
+		case "buildtimeout":
```

Comment:
please update the unit test for setConfiguration to test buildTimeout too

Reply:
💯 
Done